integration-tests/http/src/main/resources/jsse/readme.adoc

```diff
@@ -0,0 +1,4 @@
+Server keystore has to contain server certificate. It is possible to use self-signed certificate created by following command:
+
+`keytool -genkeypair -keystore keystore.p12 -storetype PKCS12 -storepass changeit -alias localhost -keyalg RSA -keysize 2048 -validity 99999 -dname "CN=localhost"
+.crt.pem -storetype JKS -keystore server.truststore -storepass changeit`
```

Comment:
Can you keep the README in the root of `integration-tests/http` instead of in the `resources` directory.

Reply:
My intention was to describe , how to create certificate  so I've put the readme in the same folder as certificate. But I agree that root makes more sense - I'll fix it